.github/workflows/build_and_test_upstream_ci_image.yml

```diff
@@ -30,4 +30,4 @@ jobs:
         continue-on-error: false
         run: |
           mkdir test_operator_work_dir output_logs
-          podman run -it -v $PWD:/project/operator-test-playbooks -v ./Dockerfiles/ci/example-metadata-without-alm-annotations:/project/operator_dir -v ./output_logs:/project/output -v ./test_operator_work_dir:/project/test_operator_work_dir -e TEST_NAME=test_for_report_failed_empty_alm_examples -e OPERATOR_DIR=/project/operator_dir -e WORK_DIR=/project/output -e OPERATOR_WORK_DIR=/project/test_operator_work_dir/ upstream_ci_image:latest
+          podman run -it -v $PWD:/project/operator-test-playbooks -v ./Dockerfiles/ci/example-metadata-wrong-channel:/project/operator_dir -v ./output_logs:/project/output -v ./test_operator_work_dir:/project/test_operator_work_dir -e TEST_NAME=test_for_report_failed_empty_alm_examples -e OPERATOR_DIR=/project/operator_dir -e WORK_DIR=/project/output -e OPERATOR_WORK_DIR=/project/test_operator_work_dir/ upstream_ci_image:latest
```

Comment:
I would keep even test-case for `example-metadata-without-alm-annotations`.
And we need to ensure it passed certification.

Reply:
I changed the test case back to test `example-metadata-without-alm-annotations` as a positive test.
I decided against adding the new test case as there's no support currently for running different tests on different metadata, and adding it in this PR would be out of scope.

Reply:
ty